CMakeLists.txt

```
@@ -256,6 +256,16 @@ if(    yaml-cpp_FOUND
     INTERFACE_INCLUDE_DIRECTORIES "${YAML_CPP_INCLUDE_DIR}"
   )
 endif()
+# Workaround to fix a problem with the imported target name
+# The target name includes the namespace yaml-cpp begining
+# from version 0.8.0. So the new exported target is now
+# yaml-cpp::yaml-cpp.
+# To work also with previous versions of yaml-cpp an alias
+# is created and the new target name is used in the project
+if(yaml-cpp_FOUND AND TARGET yaml-cpp)
+  message(STATUS "Creating target alias yaml-cpp::yaml-cpp")
```

```suggestion
  message(VERBOSE "Creating target alias yaml-cpp::yaml-cpp")
```

I don't think this message is of interest to most normal users in most cases?

---

The message can be removed completely. I add it for debugging.

---

Put your suggestion.